mockwebserverplus/src/main/java/com/orhanobut/mockwebserverplus/DelayedDispatcher.java

```diff
@@ -1,26 +0,0 @@
-package com.orhanobut.mockwebserverplus;
-
-import java.util.concurrent.TimeUnit;
-import java.util.logging.Logger;
-
-import okhttp3.mockwebserver.MockResponse;
-import okhttp3.mockwebserver.QueueDispatcher;
-import okhttp3.mockwebserver.RecordedRequest;
-
-/**
- * Delays {@link okhttp3.mockwebserver.MockWebServer} response for specified seconds.
- */
-public class DelayedDispatcher extends QueueDispatcher {
```

Comment:
MockWebServer.delayBody actually works different than this dispatcher. This was used to simulate connection timeout. `delayBody` request happens immediately and it simulates read timeout. 

```
request -> connection (timeout, delayed dispatcher) -> read (timeout, delayBody)
```

However, with anonymous dispatcher, client can just create an anonymous dispatcher and add the delay themselves.   `Thread.sleep(delayMillis)`. Therefore it's OK to remove it. Less feature, less maintenance 👍